AE_PunktBeschriftung/build/Makefile

```diff
@@ -67,10 +67,10 @@ RM = /usr/local/Cellar/cmake/3.18.4/bin/cmake -E rm -f
 EQUALS = =
 
 # The top-level source directory on which CMake was run.
-CMAKE_SOURCE_DIR = "/Users/thomasklein/Uni/11_Semester/Algorithm Engineering/AlgorithmEngineering/AE_PunktBeschriftung"
+CMAKE_SOURCE_DIR = /Users/thomasklein/Uni/11_Semester/Algorithm_Engineering/AlgorithmEngineering/AE_PunktBeschriftung
```

Comment:
I just noticed that cmake writes the absolute path on my system into the makefile, we will have to fix that :D don't know how that can be done, though

Reply:
Maybe [CMAKE_USE_RELATIVE_PATHS](https://cmake.org/cmake/help/v3.0/variable/CMAKE_USE_RELATIVE_PATHS.html) can be set to true?
I don't know much about cmake tho, so could be totally wrong.

Reply:
Apparently, that variable has been discontinued :( as far as my understanding goes, the idea of Cmake is to always generate the perfect, system-dependent makefile for a project, _instead of_ having one makefile. So it was a mistake on my part to upload the makefile in the first place, I'll remove it again and ask next week if it's okay to just use cmake instead of make. Sounded like everybody would be on board with that